debian/acrn-devicemodel.lintian-overrides

```diff
@@ -0,0 +1,2 @@
+# manpages are missing in upstream
```

Comment:
This is not only a acrn-dm problem!. If we'd support -v (--version) and  proper -h (--help) command line options for all binaries/tools involve we also could generated simple manpages using help2man automatically to avoid the linitian overrides.